ampproject/amp-wp
# Update packages and fix tests

The updated WP JS packages have been released today, including some of the fixes I've been working on regarding the build tooling.

Now anyone should be able to run the tests without having to run `npm link` or similar.

To verify:

1. Remove `node_modules`
1. Run `npm install`
1. Run `npm run test:js`

Expected output:

```
// ...

Test Suites: 1 skipped, 25 passed, 25 of 26 total
Tests:       6 skipped, 84 passed, 90 total
Snapshots:   6 passed, 6 total
Time:        22.123s
Ran all test suites.
```

Now that this is fixed we can look into running these tests on CI too.

Closes #2361.
